pkg/apis/events/v1beta1/cloudpubsubsource_types.go

```diff
@@ -116,8 +116,10 @@ func CloudPubSubSourceEventSource(googleCloudProject, topic string) string {
 }
 
 const (
-	// CloudPubSubSource CloudEvent type
-	CloudPubSubSourcePublish = "com.google.cloud.pubsub.topic.publish"
+	// ref: https://github.com/googleapis/google-cloudevents/blob/master/proto/google/events/cloud/pubsub/v1/events.proto#L26
+	// TODO: somehow reference the proto value directly.
+	CloudPubSubSourceMessagePublishedEventType = "google.cloud.pubsub.topic.v1.messagePublished"
+	CloudPubSubSourceEventDataSchema           = "https://github.com/googleapis/google-cloudevents/blob/master/proto/google/events/cloud/pubsub/v1/events.proto"
```

Comment:
shall we point to the row schemas links? https://raw.githubusercontent.com/googleapis/google-cloudevents/master/proto/google/events/cloud/pubsub/v1/events.proto

Reply:
We haven't really discussed this but I think you're right.